swagger.yaml

```diff
@@ -75,6 +77,78 @@ paths:
             text/plain:
               schema:
                 type: string
+  /projects/{id}/pdf:
```

Comment:
change `{id}` to `{projectId}`, it's possible in rest endpoints paths to have multiple ids e.g. `/users/{userId}/.../resource/{resourceId}